sources/packages/web/src/views/aest/StudentDetails.vue

```diff
@@ -0,0 +1,116 @@
+    >
+  </h5>
+  <PDStatus :pdStatus="studentDetail.pdStatus" />
+  <full-page-container>
+    <h2 class="color-blue padding-bottom-30">Student Details</h2>
+    <formio formName="studentProfileSummary" :data="initialData"></formio>
+    <div class="category-header-large">Student Appliations</div>
+    <v-col>
+      <DataTable
+        :autoLayout="true"
+        :value="studentDetail.applications"
+        class="p-m-4"
+      >
+        <Column field="applicationNumber" header="Application #"> </Column>
+        <Column field="applicationName" header="Study Period">
+          <template #body="slotProps">
+            <v-btn
+              plain
+              color="primary"
+              @click="goToApplication(slotProps.data.id)"
+              >{{ slotProps.data.applicationName }}</v-btn
+            >
+          </template></Column
+        >
+        <Column field="studyStartPeriod" header="Study Period">
+          <template #body="slotProps">
+            <span>
+              {{ dateString(slotProps.data.studyStartPeriod) }} -
+              {{ dateString(slotProps.data.studyEndPeriod) }}
+            </span>
+          </template></Column
+        >
+        <Column field="award" header="Award"></Column>
+        <Column field="status" header="Status">
+          <template #body="slotProps">
+            <Status :statusValue="slotProps.data.status" />
+          </template>
+        </Column>
+      </DataTable>
+    </v-col>
+  </full-page-container>
+</template>
+
+<script lang="ts">
+import { onMounted, ref } from "vue";
+import { useRouter } from "vue-router";
+import { AESTRoutesConst } from "@/constants/routes/RouteConstants";
+import { StudentDetail, ApplicationStatus } from "@/types";
+import { AestService } from "@/services/AestService";
+import { useFormatters } from "@/composables";
+import Status from "@/views/student/ApplicationStatus.vue";
+import PDStatus from "@/views/student/StudentPDBanner.vue";
+import FullPageContainer from "@/components/layouts/FullPageContainer.vue";
+import formio from "@/components/generic/formio.vue";
+export default {
+  components: { Status, FullPageContainer, PDStatus, formio },
+  props: {
+    studentId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup(props: any) {
+    const router = useRouter();
+    const studentDetail = ref({} as StudentDetail);
+    const initialData = ref();
+    const { dateString } = useFormatters();
+    const goBack = () => {
+      router.push({
+        name: AESTRoutesConst.SEARCH_STUDENTS,
+      });
+    };
+
+    onMounted(async () => {
+      studentDetail.value = await AestService.shared.getStudentDetail(
+        props.studentId,
+      );
+      initialData.value = {
+        firstName: studentDetail.value.firstName,
+        lastName: studentDetail.value.lastName,
+        gender: studentDetail.value.gender,
+        email: studentDetail.value.email,
+        dateOfBirth: dateString(studentDetail.value.dateOfBirth),
```

Comment:
Please ensure that this DOB is displayed in the same way as the Student Profile.